environment.yml

```diff
@@ -0,0 +1,15 @@
+name: lp
+channels:
+  - conda-forge
+dependencies:
+  - bison=3.4
+  - cmake
+  - flake8
+  - llvmdev=11.0.1
+  - make
+  - numpy
+  - pip
+  - python=3.10.2
```

Comment:
I don't think its a good idea to write a specific python version here. `conda` should be able to figure out on its own what version of python should be installed on a specific system.

Reply:
I don't think we need python in `environment.yml`

Reply:
I see. So, how will someone update the reference tests i.e., `./run_tests.py -u`? By using system Python? I don't usually prefer using system Python when I am already creating a conda environment. Not sure about others.

Reply:
I think `conda` automatically adds `python` while creating an environment.